dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGVisualizeLauncher.java

```diff
@@ -0,0 +1,286 @@
+package org.checkerframework.dataflow.cfg;
+
+import com.sun.tools.javac.file.JavacFileManager;
+import com.sun.tools.javac.main.JavaCompiler;
+import com.sun.tools.javac.util.Context;
+import com.sun.tools.javac.util.List;
+import com.sun.tools.javac.util.Options;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.Map;
+import javax.tools.JavaFileManager;
+import javax.tools.JavaFileObject;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.analysis.Analysis;
+import org.checkerframework.dataflow.analysis.Store;
+import org.checkerframework.dataflow.analysis.TransferFunction;
+import org.checkerframework.dataflow.cfg.CFGProcessor.CFGProcessResult;
+
+/**
+ * Launcher to generate the DOT or String representation of the control flow graph of a given method
+ * in a given class.
+ *
+ * <p>Usage: Directly run it as the main class to generate the DOT representation of the control
+ * flow graph of a given method in a given class. See {@link
+ * org.checkerframework.dataflow.cfg.playground.ConstantPropagationPlayground} for another way to
+ * use it.
+ */
+public class CFGVisualizeLauncher {
+
+    /**
+     * The main entry point of CFGVisualizeLauncher.
+     *
+     * @param args the passed arguments
+     */
+    public static void main(String[] args) {
```

Comment:
There isn't any documentation here on what args can be passed.  Maybe just add a reference to the `printUsage` method here?